axelrod/tournament.py

```diff
@@ -234,20 +244,18 @@ def _write_interactions_to_dict(self, results):
                     self.interactions_dict[index_pair] = [interaction]
                 self.num_interactions += 1
 
-    def _run_parallel(self, processes: int=2, progress_bar: bool = False
-                      ) -> bool:
+    def _run_parallel(self, processes: int=2) -> bool:
         """
         Run all matches in parallel
 
         Parameters
         ----------
 
-        progress_bar : bool
-            Whether or not to update the tournament progress bar
+        processes : int
+            How many processes to use.
         """
         # At first sight, it might seem simpler to use the multiprocessing Pool
```

Comment:
curiosity got the better of me.  i just checked, and pickle can now handle bound methods (in py3.6.1)!  (after the last set of work, seeing improvements to pickle fills me with joy.) I don't know about earlier versions of 3.  so perhaps leave the updated comment as the bound-method information is either out-of-date, or soon to be out-of-date? (i can't imagine anyone going in to change this again before py3.6 becomes an older revision)

here's the code i ran to check.

```
import pickle
import axelrod as axl

tournament = axl.Tournament(players=[axl.CyclerCCCDCD(), axl.Defector()])

bm_1 = tournament.play
print('original bound method: ', bm_1)
result_set_1 = bm_1(progress_bar=False)

bm_2 = pickle.loads(pickle.dumps(bm_1))
print('second bound method: ', bm_2)
result_set_2 = bm_2(progress_bar=False)

bm_3 = pickle.loads(pickle.dumps(bm_2))
print('third bound method: ', bm_3)
result_set_3 = bm_3(progress_bar=False)

print(result_set_1 == result_set_2 == result_set_3)
```

results:

```
E:\work\environments\axelrodtest\Scripts\python.exe C:/Users/Administrator/.PyCharm2017.1/config/scratches/scratch_6.py
original bound method:  <bound method Tournament.play of <axelrod.tournament.Tournament object at 0x0000000002765C50>>
second bound method:  <bound method Tournament.play of <axelrod.tournament.Tournament object at 0x0000000002DACE48>>
third bound method:  <bound method Tournament.play of <axelrod.tournament.Tournament object at 0x0000000006D08C88>>
True

Process finished with exit code 0
```